src/components/Routes/Routes.tsx

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
-import { Redirect, Route, Switch } from 'react-router-dom';
+import { Redirect, Switch } from 'react-router-dom';
+import { CompatRoute } from 'react-router-dom-v5-compat';
 import { TraceExplorationPage } from '../../pages/Explore';
 import { PLUGIN_BASE_URL, ROUTES } from 'utils/shared';
 
 export const Routes = () => {
   return (
     <Switch>
-      <Route path={prefixRoute(`${ROUTES.Explore}`)} component={TraceExplorationPage} />
+      <CompatRoute path={prefixRoute(`${ROUTES.Explore}`)} component={TraceExplorationPage} />
       <Redirect to={prefixRoute(ROUTES.Explore)} />
     </Switch>
   );
```

Comment:
I checked and it doesn't seem like we need this file or react-router here at all. We can just render `TraceExplorationPage` inside `App` instead of the `Routes` and it should work. The actual plugin routes are registered via plugin.json's `includes` property,  so I think this explicit route declaration is redundant.